File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    table = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for idx, word in enumerate(table):
        if word in s:
            s = s.replace(word, str(idx))
    return int(s)

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    const numberStrings = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine'];
    
    for (index in numberStrings) {
        const numberString = numberStrings[index];
        
        s = s.replaceAll(numberString, index)
    }
    
    return Number(s)
}